src/deluge/dsp/compressor/rms_feedback.cpp

```diff
@@ -110,14 +110,14 @@ float RMSFeedbackCompressor::calcRMS(StereoSample* buffer, uint16_t numSamples)
 	q31_t offset = 0; // to remove dc offset
 	float lastMean = mean;
 	do {
-		q31_t l = thisSample->l - hpfL.doFilter(thisSample->l, a);
-		q31_t r = thisSample->r - hpfL.doFilter(thisSample->r, a);
+		q31_t l = thisSample->l - hpfL.doFilter(thisSample->l, hpfMovability_);
+		q31_t r = thisSample->r - hpfL.doFilter(thisSample->r, hpfMovability_);
 		q31_t s = std::max(std::abs(l), std::abs(r));
-		sum += multiply_32x32_rshift32(s, s) << 1;
+		sum += multiply_32x32_rshift32(s, s);
 
 	} while (++thisSample != bufferEnd);
 
-	float ns = float(numSamples);
+	float ns = float(numSamples) / 0.5;
```

Comment:
nit: this should probably be *2 for clarity and in case the compiler does something dumb